src/window.rs

```diff
@@ -132,6 +146,41 @@ impl Window {
         window
     }
 
+    /// Create the control panel
+    fn create_control_panel(window: &winit::window::Window) -> WebView {
+        let size = window.inner_size();
+        let size = Size2D::new(size.width as i32, size.height as i32);
+        WebView::new_panel(DeviceIntRect::from_size(size))
```

Comment:
This doesn't quite work in the second window, `PipelineNamespace::install(PipelineNamespaceId(0))` will panic, I'm not sure what does `PipelineNamespace` do